weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -84,9 +97,11 @@ export const RunsTable: FC<{
   spans: SpanWithFeedback[];
   clearFilters?: null | (() => void);
 }> = ({loading, spans, clearFilters}) => {
-  const showIO = useMemo(() => {
-    return Array.from(new Set(spans.map(span => span.name))).length === 1;
+  const showIO = true;
```

Comment:
I thought I would wait and remove the rest of this once people have tried the proposed UX change out.